src/App.tsx

```diff
@@ -1,39 +1,65 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+type State = {
+  clockName: string;
+  displayClock: boolean;
```

Comment:
its should be named like hasClock its was in README file,but I think its also good name